packages/next/build/index.ts

```diff
@@ -1531,14 +1525,10 @@ export default async function build(
           pages: combinedPages,
           outdir: path.join(distDir, 'export'),
           statusMessage: 'Generating static pages',
-          exportPageWorker: sharedPool
-            ? staticWorkers.exportPage.bind(staticWorkers)
-            : undefined,
-          endWorker: sharedPool
-            ? async () => {
-                await staticWorkers.end()
-              }
-            : undefined,
+          exportPageWorker: staticWorkers.exportPage.bind(staticWorkers),
+          endWorker: async () => {
+            await staticWorkers.end()
```

Comment:
Looks like we might need to ensure `staticWorkers.end` is still called when we aren't prerendering any pages